### PR TITLE
registered new FormType and added method to createPage to be able to …

### PR DESCRIPTION
**Pull request by mattagohni**
…choose product by name in behat-tests

### CODE REVIEW

**Comment by mattagohni** [comment]
bildschirmfoto 2018-03-07 um 15 58 50

**Comment by michz** [comment]
I'm excited about how this will perform in usability when there are a few thousand products...

File: src/Resources/config/resources.yml [hunk 1]
```diff
@@ -3,6 +3,7 @@ sylius_resource:
         solutiondrive.product_bundle:
             classes:
                 model: SolutionDrive\SyliusProductBundlesPlugin\Entity\ProductBundle
+                form: SolutionDrive\SyliusProductBundlesPlugin\Form\Type\ProductBundleType
```
Comment (michz) [inline src/Resources/config/resources.yml:6]: In my opinion we should switch in this plugin the base namespace to `solutionDrive` as defined by corporate identity.

**Review by michz — DISMISSED** [review]

**Review by Jnoack331 — DISMISSED** [review]

**Review by Jnoack331 — APPROVED** [review]

File: src/Resources/config/resources.yml (hunk 1 above)
Comment (mattagohni) [reply src/Resources/config/resources.yml:6]: I created an issue for this:
#19

**Review by sd-lueckel — APPROVED** [review]

**Pull request merged**